src/cocotb_tools/makefiles/simulators/Makefile.icarus

```diff
@@ -70,10 +70,12 @@ endif
 
 # Compilation phase
 
+ICARUS_PLUSARGS = $(call deprecate,PLUSARGS,COCOTB_PLUSARGS)
```

Comment:
This warns early. Instead do the same thing as the other variables and don't use PLUSARGS to add `-fst`.